ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -17835,6 +17806,33 @@ ABoundActivity * HqlCppTranslator::doBuildActivityHTTP(BuildCtx & ctx, IHqlExpre
             buildTransformBody(onFailCtx, onFail->queryChild(0), dataset, NULL, expr, selSeq);
         }
     }
+    //virtual unsigned getFlags()
```

Comment:
Not for this PR, but this looks like a lot of duplicated code which should really be commoned up with the soapcall code - there much in common between  doBuildActivitySOAP and doBuildActivityHTTP